Lib/test/libregrtest/runtest_mp.py

```diff
@@ -149,8 +149,6 @@ def run(self):
 def run_tests_multiprocess(regrtest):
     output = queue.Queue()
     pending = MultiprocessIterator(regrtest.tests)
-    test_timeout = regrtest.ns.timeout
```

Comment:
I don't think we want to remove this in this PR. If there's a problem with parallel tests and the `--timeout` option, better open a separate issue about it.

Reply:
I removed the commit from the PR and I will open another one for this change.